cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -778,7 +780,7 @@ func main() {
 		}
 	}
 
-	isDebugCmd := *debugList || *debugIndex != "" || *debugShard != "" || *debugMeta != "" || *debugMerge
+	isDebugCmd := *debugList || *debugIndex != "" || *debugShard != "" || *debugMeta != "" || *debugMerge || *debugFind != "" || *debugListIndexed
```

Comment:
next time we chat lets talk about what we can do to improve our sprawl of debug stuff :)

Reply:
Yep, this is getting out of control. We could introduce a subcommand and move all the debug commands to a separate file. This should clean up things nicely. I use all of commands, so I don't want to get rid of them.